src/containers/Triggers/Triggers.js

```diff
@@ -93,12 +94,7 @@ function Triggers(props) {
   }));
 
   return (
-    <ListPageLayout
-      {...props}
-      error={getError()}
-      filters={filters}
-      title="Triggers"
-    >
+    <ListPageLayout error={getError()} filters={filters} title="Triggers">
```

Comment:
Nit: Shouldnt this title be translated?

Reply:
I see similar titles for ListPageLayout elsewhere, so I'm guessing these are being treated as special keywords and thus not translated?

Reply:
exactly, we don't translate the CRD names, e.g. `PipelineRun`, `Trigger`, etc.